### DISCUSSION TITLE
Touch events ReactJS

### DISCUSSION THREAD

**Discussion by magikarp28** [matteobruni/tsparticles#4837] [Q&A] [answered]
There seems only to be click and hover events for interactivity. Is there a way to use touch events so it works with multitouch?
Thanks.

**Comment by matteobruni** [comment, marked as answer]
> There seems only to be click and hover events for interactivity. Is there a way to use touch events so it works with multitouch?
> Thanks.

`onClick` and `onHover` configurations already catch touch events

**Reply by magikarp28** [reply]
Those are not touch events and therefore only work for a single touch, I need multiple "hover" events at the same time.

**Reply by magikarp28** [reply]
I thought only mouse events were used but I do see references to touch events. Well they certanly don't work on windows or android devices.

**Reply by matteobruni** [reply]
Multi touch events are not handled, only the last touch will be considered. This could be a feature request.

**Reply by magikarp28** [reply]
Okay, that explains it, thanks. It would certanly be very nice if there was an option to handle multitouch.
